README.md

```diff
@@ -75,6 +75,7 @@ The following settings are supported:
 * `quarkus.tools.alwaysShowWelcomePage` : Determines whether to show the welcome page on extension startup. Default is `true`.
 * `quarkus.tools.starter.api` : Quarkus API base URL. Default is `https://code.quarkus.io/api`.
 * `quarkus.tools.starter.showExtensionDescriptions`: Determines whether to show the Quarkus extension descriptions when selecting Quarkus extensions. Default is `true`.
+* `qute.trace.server`: Trace the communication between VS Code and the Qute language server in the Output view. Default is `off`.
```

Comment:
I think we should have a `Qute support` section in the README and add this settings in the Qute support section. @fbricon any though?

Reply:
Issue opened at https://github.com/redhat-developer/vscode-quarkus/issues/427

Reply:
OK lets merge the pr. I will do the change in the documentation pr.